lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt

```diff
@@ -58,6 +62,8 @@ internal class AstToLogicalVisitorTransform(
 
         var (select, algebraAfterAggregation) = transformAggregations(node, algebra) ?: node to algebra
 
+        select = removeCallAggregations(callAggregationReplacer, select)
```

Comment:
The "Call Aggregations" should never be considered "grouping operator aggregation functions". Their related insofar as they have a similar name and computation, but a "Call Aggregation" shouldn't be conflated with traditional relation-level aggs